action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
     description: 'Unity License Client `services-config.json` encoded base base64 string. Required for `Floating` license activations.'
     required: false
     default: ''
+  license-version:
+    description: 'Must be one of `5.x` (Unity 5.x), `6.x` (Unity 2017+)'
```

Comment:
I think we can automatically detect this based on the unity editor version we're trying to activate for.